kclvm/cmd/src/tests.rs

```diff
@@ -559,3 +559,39 @@ fn test_plugin_not_found() {
         Err(msg) => assert!(msg.contains("the plugin package `kcl_plugin.not_exist` is not found, please confirm if plugin mode is enabled")),
     }
 }
+
+#[test]
+fn test_error_message_fuzz_matched() {
```

Comment:
Move these test cases into the resolver crate `kcl/kclvm/sema/src/resolver/tests.rs'`